system/addon-manifest.xml

```diff
@@ -9,6 +9,7 @@
   <addon>kodi.binary.global.network</addon>
   <addon>kodi.binary.global.gui</addon>
   <addon>kodi.binary.global.filesystem</addon>
+  <addon optional="true">kodi.binary.global.tools</addon>
```

Comment:
Why optional?

Reply:
Thought to prevent problems, if them not as dummy addon in Kodi.
Also part not work direct with Kodi. 
Can also make not optional.

Reply:
I would consider making mandatory, unless there's some way or reason to configure kodi without this

Reply:
Have changed.